src/components/CopyLink.js

```diff
@@ -46,18 +51,36 @@ export const urlParamsToViewTarget = (url) => {
   // Returns: Object (viewTarget) - see App.js
   // Source for parsing: https://stackoverflow.com/questions/8648892/how-to-convert-url-parameters-to-a-javascript-object
 
-  // Get portion of URL after '?'
-  const params = new URL(url).searchParams;
-  if (params.toString() === "") return null;
-  // Parse the parameters from the URL
-  const urlParams = new URLSearchParams(params);
-  const entries = urlParams.entries();
-  const result = {};
-  for (const [key, value] of entries) {
-    // each 'entry' is a [key, value] tuple
-    if (value !== undefined) {
-      result[key] = value;
+  var result = {};
+  const s = url.href.split('?');
+  // If url contains information on viewObject
+  if (s[1]) {
+    result = qs.parse(s[1]);
+  } else {
+    // Get portion of URL after '?'
+    const params = new URL(url).searchParams;
+
+    if (params.toString() === "") return null;
+    // Parse the parameters from the URL
+    const urlParams = new URLSearchParams(params);
+    const entries = urlParams.entries();
+    for (const [key, value] of entries) {
+      console.log("key: ", key);
+      console.log("value: ", value);
+      // each 'entry' is a [key, value] tuple
+      if (value !== undefined) {
+        // If multiple values under the same key is recieved
+        if (Object.prototype.hasOwnProperty.call(result, key)) {
+          if (typeof result[key] === 'string') {
+            result[key] = [result[key]];
+          }
+          result[key].push(value);
+        } else {
+          result[key] = value;
+        }
+      }
```

Comment:
I think that, when we split the URL on `?` and we don't have anything after a first `?`, then the URL's `searchParams` are also always empty, because if we had them they would be after `?`. So I think we would always hit the `return null` in the `else` and never hit most of the logic in the `else`.

Also, we don't need this whole block, because `qs.parse()` just implements all this stuff for us, right?

We could probably simplify this down to:

```suggestion
  const params = new URL(url).searchParams;
  if (params) {
    result = qs.parse(params);
  }
```

Is there a reason why that wouldn't work?